pogobuf/pogobuf.client.js

```diff
@@ -163,6 +163,15 @@ function Client() {
     this.setResponseCallback = function(callback) {
         self.on('raw-response', callback);
     };
+    
+    /**
+     * Enables or disables automatic Long.js object conversion in RPC responses
```

Comment:
Maybe expand a little: Enables or disables automatic conversion of Long.js to primitive types in API response objects.